package.json

```diff
@@ -38,23 +42,19 @@
     "@babel/plugin-proposal-class-properties": "^7.12.13",
     "@babel/plugin-transform-classes": "^7.12.13",
     "babel-preset-gatsby": "^1.14.0",
+    "cypress": "^9.4.1",
+    "cypress-wait-until": "^1.7.2",
     "eslint": "^7.32.0",
     "eslint-config-airbnb": "^18.2.1",
+    "eslint-plugin-cypress": "^2.12.1",
     "eslint-config-prettier": "^8.3.0",
     "eslint-plugin-import": "^2.24.0",
     "eslint-plugin-jsx-a11y": "^6.4.1",
     "eslint-plugin-react": "^7.24.0",
+    "http-server": "^14.1.0",
     "husky": "^7.0.2",
     "lint-staged": "^11.1.2",
-    "prettier": "^2.3.2"
-  },
-  "lint-staged": {
-    "*.js": [
-      "eslint --fix",
-      "prettier --write"
-    ],
-    "*.{css,scss,json,html,yaml,md,mdx}": [
-      "prettier --write"
-    ]
```

Comment:
is this lint-staged deletion intentional?

Reply:
yes, I moved it to `.lintstagerc` to be consistent with the other config files at the root of the project.